src/condvar.rs

```diff
@@ -198,8 +219,11 @@ impl Condvar {
                 }
                 TOKEN_NORMAL
             };
-            parking_lot_core::unpark_requeue(from, to, validate, callback);
+            let res = parking_lot_core::unpark_requeue(from, to, validate, callback);
+
+            res.unparked_threads
```

Comment:
This will only ever be 1 or 0 since only 1 thread is unparked. The rest are requeued onto the mutex.

Reply:
Sorry about that, should be fixed now. I added a new field to `UnparkResult` as well. Do you want me to seal the type to make it future proof?

Reply:
What do you mean by "seal the type"?

Reply:
Add a private field so the struct can't be exhaustively matched.

Reply:
Ah yes, please do that.